src/analytics/config.py

```diff
@@ -21,7 +21,7 @@ class Config:
     TESTING = False
     CSRF_ENABLED = True
     SECRET_KEY = env_var("SECRET_KEY")
-    GRID_URL = "https://platform.airqo.net/api/v2/devices/measurements/grids/"
+    GRID_URL = os.getenv("GRID_URL_ID")
```

Comment:
Hi @wabinyai , thanks for this.

- We might want to remove this from here since we are already describing it below under different environments.
- We might also want to crosscheck that the respective .ENVs have been updated accordingly. I can help with that if you can share them via my Slack DM.

Reply:
Resolved in the new PR

Reply:
Thanks @wabinyai .....

- We might also want to crosscheck that the respective new ENVs have been updated accordingly. I can help with that if you can share them via my Slack DM.

Reply:
The analytics .env 
the update should be

BIGQUERY_HOURLY_CONSOLIDATED, 
GRID_URL_ID

Reply:
Alright @wabinyai , thanks!

Reply:
Hi @wabinyai :
Are the following ENVs the new ENVs you have introduced in this PR?

- BIGQUERY_HOURLY_CONSOLIDATED
- BIGQUERY_HOURLY_CONSOLIDATED_PR0D
- BIGQUERY_HOURLY_CONSOLIDATED_STAGE
- BIGQUERY_HOURLY_CONSOLIDATED_DEV
- GRID_URL_ID
- STAGE_GRID_URL_ID
- DEV_GRID_URL_ID

If YES,

1. where the PROD ENV for GRID_URL_ID? 
2. Where are these ENVs being called from in the codebase? Can you highlight those sections using new review comments?

Reply:
Resolved back to the previous one